nss-day-cohort-41/react-nutshell-almonds
# Jrr nutshell

Welcome message with user profile and name will populate on the home page. Friend's list images populated below and should differ based on session storage id. Message layout to match Triggs changes and simple css added to improve visibility. 

Changes made to:
Application Views
User Card
User List
User Edit Form
Friend Card
Friend List
Article List
Article Card
Event List 
Event Card

When testing please note any bugs that may arise. 


Type of change


 New feature (non-breaking change which adds functionality)
Testing Instructions
Please fetch my branch jrr-nutshell and npm start to ensure that changes made did not break any of your code. Also note that your database may need to be updated to visually see some changes. 

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
